axelrod/plot.py

```diff
@@ -59,3 +61,33 @@ def payoff(self):
             return figure
         else:
             return None
+
+    def stackplot(self, populations):
+        if self.matplotlib_installed:
+            figure, ax = plt.subplots()
+            turns = range(len(populations))
+            pops = [[populations[iturn][ir] for iturn in turns] for ir in self.result_set.ranking]
+            ax.stackplot(turns, *pops)
```

Comment:
Does stackplot automatically sort these by the final score or is this happening somewhere else? (The output looks sorted but perhaps that's not happening).